neo4j-jdbc/src/main/java/org/neo4j/jdbc/StatementImpl.java

```diff
@@ -149,7 +148,14 @@ protected final int executeUpdate0(String sql, boolean applyProcessor, Map<Strin
 		return transaction.runAndDiscard(sql, getParameters(parameters), this.queryTimeout, transaction.isAutoCommit())
 			.resultSummary()
 			.map(ResultSummary::counters)
-			.map(SummaryCounters::totalCount)
+			.map(c -> {
+				var rowCount = c.nodesCreated() + c.nodesDeleted();
+				if (rowCount == 0 && c.containsUpdates()) {
+					var labelsAndProperties = c.labelsAdded() + c.labelsRemoved() + c.propertiesSet();
+					rowCount = (labelsAndProperties > 0) ? 1 : 0;
```

Comment:
Just out of curiosity, is this just a best-effort mapping we can do? I assume labels of multiple rows could be modified.

Also, just wanted to ask if we want to include relationships and some other properties of `SummaryCounters`?